index.html

```diff
@@ -91,15 +93,26 @@ <h4>Spring 2022</h4>
                 <input type="email" value="" name="EMAIL" placeholder="Enter your email"/>
                 <button form="intro-form" type="submit">Subscribe</button>
               </form>  -->
-       
+
           </div>
         </div>
-        <div class="button-container">
-         
-        <a href = "https://forms.gle/1BmGY1JetzMp1cLP6 " target="_blank" class="ugly-bart-fix-fix">
-          <div class="bart-fix">Sign our interest form for more info</div>
-      </a>
-      </div>
+
+
+        <div class="landing-page-info" style="margin-top: 5rem;">
+          <div class="landing-page-about">
+            <h4>We're hard at work getting PickHacks 2022 underway</h4>
+            <h5>In the meantime, fill out this interest form to stay in the loop!</h5>
+          </div>
+        </div>
+        <!--The seperation is to seperate text and button vertically as opposed to horizontally-->
+        <div class="landing-page-info">
+          <div class="landing-page-about" style="margin-top: 2rem; margin-bottom: 5rem;">
+            <a href="https://forms.gle/1BmGY1JetzMp1cLP6">Interest Form</a>
```

Comment:
Attach a `target="_blank"` to the <a></a> element. This will make the form open in a new tab.